_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
So the `usa-site-footer` is just for the actual styleguide?

My one concern here is if users look at the actual markup on the styleguide to learn how to do things and they come across a class like that.